lib/features/create_transaction/screens/choose_amount_slider_screen.dart

```diff
@@ -0,0 +1,253 @@
+// ignore_for_file: prefer_const_constructors, prefer_const_literals_to_create_immutables
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/create_transaction/cubit/create_transaction_cubit.dart';
+import 'package:givt_app_kids/features/create_transaction/models/transaction.dart';
+import 'package:givt_app_kids/features/create_transaction/screens/success_screen.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/shared/widgets/wallet.dart';
+
+import 'package:givt_app_kids/widgets/back_button.dart' as custom_widgets;
+
+class ChooseAmountSliderScreen extends StatefulWidget {
+  static const String routeName = "/choose-ammount-slider-bloc";
+
+  const ChooseAmountSliderScreen({Key? key}) : super(key: key);
+
+  @override
+  State<ChooseAmountSliderScreen> createState() =>
+      _ChooseAmountSliderScreenState();
+}
+
+class _ChooseAmountSliderScreenState extends State<ChooseAmountSliderScreen> {
+  late final ProfilesCubit _profilesCubit;
+
+  @override
+  void initState() {
+    _profilesCubit = context.read<ProfilesCubit>();
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Probably you can extract some widgets from this build method or create some methods. It seems a bit too big

Reply:
you are right. Just for the context - the story has level 3 of detalisation, means w/o design. That's why we decided to re use existing UI as much as possible, since it will be redesigned anyway.